azure-iot-device/azure/iot/device/provisioning/transport/constant.py

```diff
@@ -0,0 +1,50 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""
+This module contains constants related to the transport package.
+"""
+
+__all__ = [
+    "USER_AGENT",
+    "API_VERSION",
+    "SUBSCRIBE_TOPIC_PROVISIONING",
+    "PUBLISH_TOPIC_REGISTRATION",
+]
+
+USER_AGENT = "azure-iot-provisioning-devicesdk" + "/" + "0.0.1"
+"""
+Default interval for polling, to use in case service doesn't provide it to us.
+"""
+DEFAULT_POLLING_INTERVAL = 2000
```

Comment:
>2000 [](start = 27, length = 4)

Doesn't python use seconds as the de-facto unit of time?  This looks like you're using milliseconds.  (Or you're polling every 33 minutes).

Reply:
Yeah I was converting it in code. I can do secs here itself.

---
In reply to: [276843389](https://github.com/Azure/azure-iot-sdk-python-preview/pull/81#discussion_r276843389) [](ancestors = 276843389)